pkg/apis/core/types.go

```diff
@@ -544,6 +566,41 @@ const (
 	PersistentVolumeClaimNodeResizeFailed ClaimResourceStatus = "NodeResizeFailed"
 )
 
+// +enum
+type PersistentVolumeClaimModifyVolumeStatus string
+
+const (
+	// Pending indicates that the PersistentVolumeClaim cannot be modified due to requirements not being met, such as
```

Comment:
"cannot be modified due to unmet requirements, such as the specified VolumeAttributesClass not existing"